### PR TITLE
treewide: Modify the `wifi_cred add` command in examples

### PR DESCRIPTION
**Pull request by rado17**
`wifi_cred add` is modified now. Update the examples in samples and documentation to reflect the same.

### CODE REVIEW

File: doc/nrf/includes/wifi_credentials_shell.txt [hunk 1]
```diff
@@ -9,9 +9,10 @@ Once you have flashed your device with this sample, connect to your device's UAR
 .. parsed-literal::
    :class: highlight
 
-   wifi_cred add *NetworkSSID* *SecurityMode (OPEN, WPA2-PSK, WPA2-PSK-SHA256, WPA3-SAE)* *NetworkPassword*
+   wifi_cred add -s *NetworkSSID* -k *SecurityMode* -p *NetworkPassword*
 
-Where *NetworkSSID* is replaced with the SSID of the Wi-Fi access point you want your device to connect to, and *NetworkPassword* is its password.
+Where *NetworkSSID* is replaced with the SSID of the Wi-Fi access point you want your device to connect to, and *NetworkPassword* is its password. *SecurityMode* is replaced by the number as listed here:
+ 0:None, 1:WPA2-PSK, 2:WPA2-PSK-256, 3:SAE-HNP, 4:SAE-H2E, 5:SAE-AUTO, 6:WAPI, 7:EAP-TLS, 8:WEP, 9: WPA-PSK, 10: WPA-Auto-Personal, 11: DPP.
```
Comment (peknis) [inline doc/nrf/includes/wifi_credentials_shell.txt:15]:
```suggestion
Where *NetworkSSID* is replaced with the SSID of the Wi-Fi access point you want your device to connect to, and *NetworkPassword* is its password.
*SecurityMode* is replaced by the number as listed here:

* 0:None
* 1:WPA2-PSK
* 2:WPA2-PSK-256
* 3:SAE-HNP
* 4:SAE-H2E
* 5:SAE-AUTO
* 6:WAPI
* 7:EAP-TLS
* 8:WEP
* 9: WPA-PSK
* 10: WPA-Auto-Personal
* 11: DPP
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by krish2718 — COMMENTED** [review]
I can still few paths not updated:
```
nrf/samples/wifi/shell/README.rst
nrf/samples/cellular/nrf_cloud_multi_service/README.rst
```

**Review by simensrostad — APPROVED** [review]

**Review by sachinthegreen — APPROVED** [review]

File: doc/nrf/includes/wifi_credentials_shell.txt (hunk 1 above)
Comment (rado17) [reply doc/nrf/includes/wifi_credentials_shell.txt]: Done.

**Comment by rado17** [comment]
> I can still few paths not updated:
> 
> ```
> nrf/samples/wifi/shell/README.rst
> nrf/samples/cellular/nrf_cloud_multi_service/README.rst
> ```

Updated the complete `help` in shell sample. Updated the example in `nrf_cloud_multi_service`

File: doc/nrf/includes/wifi_credentials_shell.txt [hunk 2]
```diff
@@ -9,9 +9,24 @@ Once you have flashed your device with this sample, connect to your device's UAR
...
+* 2:WPA2-PSK-256
+* 3:SAE-HNP
+* 4:SAE-H2E
+* 5:SAE-AUTO
+* 6:WAPI
+* 7:EAP-TLS
+* 8:WEP
+* 9: WPA-PSK
+* 10: WPA-Auto-Personal
+* 11: DPP
```
Comment (peknis) [inline doc/nrf/includes/wifi_credentials_shell.txt:28]:
```suggestion
* 9:WPA-PSK
* 10:WPA-Auto-Personal
* 11:DPP
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/includes/wifi_credentials_shell.txt (hunk 2 above)
Comment (rado17) [reply doc/nrf/includes/wifi_credentials_shell.txt]: Done.

**Review by peknis — APPROVED** [review]

**Pull request merged**